about-peer-review/aims-and-scope.md

```diff
@@ -24,9 +24,12 @@ package.
 support open science. 
 
 ## Is Your Package in Scope For pyOpenSci Review?
-pyOpenSci reviews packages within a set of categories define below. 
-If you are unsure whether your package fits into one of these categories, please 
-open a [pre-submission inquiry via a GitHub Issue](LINK) to get feedback from 
+
+pyOpenSci only reviews packages that fall within our specified domain and 
+technical scope listed below. 
```

Comment:
please look lower down on the page for the technical scope section. it should look like this:

<img width="1038" alt="Screen Shot 2022-12-13 at 1 09 28 PM" src="https://user-images.githubusercontent.com/7649194/207433988-c5c76b4a-cf99-4c45-bea5-8dfec6277a52.png">

I could create a link on the page to that section?

Reply:
oh i can see how you wouldn't see that. i have that scope document as an include file to avoid making mistakes / inconsistencies with text around scope given we may want to publish the scope in a few spots. .

Reply:
Now I get it 😄. Thanks for clarifying. I like the idea of the link too.

Reply:
ok wonderful!! thanks for all of the great feedback!